core-rust/core-api-server/src/core_api/generated/models/system_version.rs

```diff
@@ -17,6 +17,8 @@ pub enum SystemVersion {
     V1,
     #[serde(rename = "V2")]
     V2,
+    #[serde(rename = "V3")]
+    V3,
```

Comment:
Side note: Slightly annoyingly, we'll also need a new Gateway API build to handle this :D

Reply:
If we need it today, give me a ping and I can try to do it.